### ISSUE TITLE
extract-inline-html parser can hang due to regexp catastrophic backtracking

### ISSUE THREAD

**Issue by mikg2003** [angular-eslint/angular-eslint#68]
If you configure an override for `*.component.ts` files to use the `extract-inline-html` parser like so:
```
{
    "files": [
        "*.component.ts"
    ],
    "parser": "@typescript-eslint/parser",
    "parserOptions": {
        "sourceType": "module"
    },
    "plugins": [
        "@angular-eslint/template"
    ],
    "processor": "@angular-eslint/template/extract-inline-html"
}
```

And if it encounters a component with some poor formatting (try adding a couple spaces or a tab before the ending of the `@Component` attribute like this):
```
@Component({
    selector: 'widget',
    templateUrl: './widget.component.html',
    styleUrls: ['./widget.component.scss']
  }) <-- space before the curly brace here
```

The linting will never complete because the regex [here](https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin-template/src/processors.ts#L4) hangs.  So unfortunately the `ng lint` execution will just sit there without any sort of error ever occurring.

**Comment by JamesHenry** [comment]
Thank you for the test case to work with as per my comment here: https://github.com/angular-eslint/angular-eslint/issues/60#issuecomment-612591044 this section of the original POC needs to be replaced with something more robust, I have a PR in the works

**Comment by mikg2003** [comment]
That's great!  Thanks for being so responsive.

**Comment by JamesHenry** [comment]
I have added it as a test case in #70

**Comment by JamesHenry** [comment]
Released `v0.0.1-alpha.28`
